src/gh_pr_upsert/cli.py

```diff
@@ -1,13 +1,97 @@
+import sys
 from argparse import ArgumentParser
 from importlib.metadata import version
+from subprocess import CalledProcessError
 
+from gh_pr_upsert import core, git
+from gh_pr_upsert.exceptions import PRUpsertError
 
-def cli(_argv=None):  # pylint:disable=inconsistent-return-statements
-    parser = ArgumentParser()
+
+def cli(_argv=None):  # pylint:disable=too-complex
+    parser = ArgumentParser(description="Create or update a GitHub pull request.")
     parser.add_argument("-v", "--version", action="store_true")
+    parser.add_argument(
+        "--base-remote",
+        help="the git remote to use for the base of the pull request (default: 'origin')",
+        default="origin",
+    )
+    parser.add_argument(
+        "--base-branch",
+        help="the git branch to use for the base of the pull request (default: --base-remote's default branch)",
+    )
+    parser.add_argument(
+        "--local-branch",
+        help="the local git branch to push (default: the current branch)",
+    )
+    parser.add_argument(
+        "--head-remote",
+        help="the git remote to use for the head of the pull request (default: 'origin')",
+        default="origin",
+    )
```

Comment:
By default the GitHub repo that the commits will be pushed to is also the `origin` repo